index.d.ts

```diff
@@ -269,6 +269,7 @@ declare module "react-native-maps" {
 
   export default class MapView extends React.Component<MapViewProps, any> {
     getCamera(): Promise<Camera>;
+    getAddress(): Promise<LatLng>;
```

Comment:
The typing for `getAdress is incorrect here. For `getAddress`, I suggest passing in a single `coordinate` object as a param instead of lat/long individually. This will match the other methods we have like `pointForCoordinate`. 

Also, we'll need a create a new type for the payload you are returning on this call. Here's an example of how that might look. I renamed `getAddress` to `addressForCoordinate` for clarity but am open to other suggestions as well.

```
type Address {
  name: string
  throughFare: string
  locality: string
  ...
}
```

`addressForCoordinate(coordinate: LatLng): Promise<Address>`